website/docs/foundations/elevation/index.md

```diff
@@ -10,6 +10,6 @@ description: Elevation can be used to offset and draw attention to content or in
 </section>
 
 <section data-tab="Specifications">
```

Comment:
@Dhaulagiri for consistency, I think this should be guidelines and not specifications

Reply:
🤔  on _components_ we would call this Specifications, but since it's not quite anatomy it's more guidelines?

Reply:
This page tells consumers where to apply each elevation style which could be categorized as usage guidelines more than specs

Reply:
Updated this to be under guidelines